hr_holidays_leave_request_wizard/models/hr_leave.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2018-2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from pytz import timezone, UTC
+from datetime import time
+
+from odoo import api, fields, models
+
+
+class HrLeave(models.Model):
+    _inherit = 'hr.leave'
+
+    @api.model
+    def _default_get_request_parameters(self, values):
+        HrEmployee = self.env['hr.employee'].sudo()
```

Comment:
Do we really need sudo here ?
(Just to be sure it doesn't disclose anything over external RPC :wink:)